test/unit/clitests.json

```diff
@@ -0,0 +1,18 @@
+{
+  "spec_dir": "test/unit",
+  "spec_files": [
+     "cff_parser_spec.js",
```

Comment:
Nit: this line and the ones below are indented with three spaces instead of two.